kitchen-tests/.kitchen.yml

```diff
@@ -7,6 +7,8 @@ driver:
 
 provisioner:
   name: chef_github
+  chef_omnibus_url: "https://omnitruck.chef.io/current/install.sh"
+  chef_omnibus_install_options: "-n"
```

Comment:
We should be using the following [per our documentation](https://docs.chef.io/packages.html#omnitruck-install-scripts):

```yaml
chef_omnibus_url: "https://omnitruck.chef.io/install.sh"
chef_omnibus_install_options: "-c current"
```

The default project is chef so adding a `-P chef` is not required but is more explicit so you could add it. The `-n` option only exists for backward compatibility and should not be used in any of our code/repos/products/documentation:
https://github.com/chef/mixlib-install/blob/master/lib/mixlib/install/generator/bourne/scripts/script_cli_parameters.sh#L24-L25

I also opened a PR in Test Kitchen to ensure it is using `https://omnitruck.chef.io/install.sh` as its default value for `chef_omnibus_url`: https://github.com/test-kitchen/test-kitchen/pull/1028